lib.nix

```diff
@@ -209,7 +209,11 @@ rec
                 pushd $out/$member > /dev/null
                 mkdir -p src
                 # Avoid accidentally pulling `std` for no-std crates.
-                echo '#![no_std]' >src/lib.rs
+                cat <<EOF >src/lib.rs
+                #![no_std]
+                #[panic_handler]
+                fn panic(_info: &core::panic::PanicInfo) -> ! { loop {} }
```

Comment:
what are these two lines for?

Reply:
If you compile with `crate-type=cdylib`, and also use `no_std`, the compilation will fail because no panic handler has been provided (the bit that unwinds the stack trace, usually provided by the standard library). This is just a dummy stub to get compilation to succeed. See https://doc.rust-lang.org/nomicon/panic-handler.html for more details.

Reply:
Thanks! Added a code suggestion; worth having the clarification near the code.